test/end-to-end/company.js

```diff
@@ -1933,4 +1933,195 @@ describe("Company endpoint", () => {
             expect(res.body).toHaveProperty("maxOffersReached", true);
         });
     });
+
+    describe("PUT /company/edit", () => {
```

Comment:
Suggestions for new tests:
- No login or god (fail)
- Blocked and disabled company (from admin and company perspective)
- Check if the company's offers update ownerName and ownerLogo. Since we don't use a relationship DB, data integration isn't as straight forward

Reply:
Added all of these, thank you!